package.json

```diff
@@ -25,6 +27,8 @@
     "postinstall": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier",
     "bootstrap": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier",
     "circleci:artifacts": "node scripts/download-vsix-from-circleci.js",
+    "commit-init": "commitizen init cz-conventional-changelog --save-dev --save-exact --force",
```

Comment:
Is this necessary for someone cloning new project? If yes can we make it part of postinstall?

Reply:
That is a good idea, I will add it.

Reply:
I decided to undo the change to postinstall. It's not required and would need a dev to install commitizen prior to init. These steps have been added to the contributing doc.

Reply:
We could do `npx commitizen ...` and dev doesn't need to install it, or add it to dev dependency. And why is this not required?

Reply:
but I agree to the point that `postinstall` is not the best place, since it might be excessive every time we add a dependency. Git doesn't have a `post-clone`, but there's a way to simulate that. https://github.com/git-hook/post-clone. The goal is to have a setup as simple as possible, but let's not worry about it now

Reply:
So it has been added as a dev dependency, but by default when you run `npm install` it is not installed. A user would need to use `npm install --dev`. 

This is not actually required because commitizen is meant more as a guide. Someone doesn't have to use it so long as their commit format matches "type: commit message", where type is a value that the husky hook expects.

Reply:
Sounds good! thanks for the explanation @sfsholden .

Reply:
@xyc You bet, thanks for the great questions